File: Data Structures & Algorithms/duplicate-integer/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        for (let i = 0; i >= nums.length -1;i++){
            for (let x = i+1; i>= nums.length;i++){
                if(nums[i] === nums[x]){
                    return true
                }
            }
        }
        return false
    }

}

File: Data Structures & Algorithms/is-anagram/submission-2.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        l ,r = 0, len(s) - 1
        while r > l:
            if s[l] == s[r]:
                l += 1
                r -= 1
            else:
                return False
        return True